content/src/denylist/DenylistServiceDecorator.ts

```diff
@@ -238,6 +238,23 @@ export class DenylistServiceDecorator implements MetaverseContentService {
     return this.service.listenToDeployments(listener)
   }
 
+  getEntitiesByIds(ids: EntityId[]): Promise<Entity[]> {
+    return this.repository.task(async (task) => {
```

Comment:
Is it necessary to run something similar to `filterDenylisted()` here?

Reply:
No, it isn't necessary.

When a pointer is denylisted, then no entities are returned for it. But when an entity is denylisted, it is returned, but the content files are not shown, and the metadata only says `'Denylisted'`. All that is done in `sanitizeEntities`, and that's why it isn't necessary to filter the entity ids